script.js

```diff
@@ -10,3 +10,79 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
   hamburger.classList.remove('active');
   navList.classList.remove('active');
 }));
+
+// Validate contact form
+/* function validate() {
+  const text = document.getElementById('text1').value;
+  const regx = /^([a-z0-9]+)@([a-z0-9-]+).([a-z]{2,20})(.[a-z]{2,8})?$/;
+
+  if (regx.test(text)) {
+    document.getElementById('ibitext').innerHTML = 'Email is valid';
+    document.getElementById('ibitext').style.visibility = 'visible';
+    document.getElementById('ibitext').style.color = 'blue';
+  } else {
+    document.getElementById('ibitext').innerHTML = 'Email requires lowercase only';
+    document.getElementById('ibitext').style.visibility = 'visible';
+    document.getElementById('ibitext').style.color = 'maroon';
+  }
+}
+validate(); */
```

Comment:
- [ ] Please try not to leave blocks of commented code. It makes the code look unclean. 👍